### PR TITLE
Add capibmadm vpc image list command

### PR DESCRIPTION
**Pull request by dharaneeshvrd**
**What this PR does / why we need it**:

Add capibmadm vpc image list command.
 
**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

/area provider/ibmcloud

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note

```

### CODE REVIEW

**Comment by dharaneeshvrd** [comment]
@mkumatag @Karthik-K-N @Prajyot-Parab 
Can you please review this PR?

File: cmd/capibmadm/cmd/vpc/image/type.go [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+			{
+				Name: "Visibility",
+				Type: "string",
+			},
+			{
+				Name: "Encryption",
+				Type: "string",
+			},
+			{
+				Name: "ResourceGroupName",
```
[the remark this answers is not in the document]
Comment (Prajyot-Parab) [reply cmd/capibmadm/cmd/vpc/image/type.go:98]: Rest LGTM /cc @Karthik-K-N

[the remark this answers is not in the document]
Comment (dharaneeshvrd) [reply cmd/capibmadm/cmd/vpc/image/type.go]: Fixed, also refactored the column names to caps.

**Review by Prajyot-Parab — APPROVED** [review]
/lgtm

**Review by mkumatag — APPROVED** [review]
/lgtm

**Pull request merged**